setup.py

```diff
@@ -32,32 +33,27 @@
     'pytest>=4.6.4,<5.0.0',
```

Comment:
```suggestion
    "pytest-invenio>=1.4.0",
```
Also - remove these packages listed bellow from ```tests_require =[]```
```
'check-manifest>=0.25',
'coverage>=4.5.3',
'isort>=4.2.2',
'mock>=1.3.0',
'pydocstyle>=3.0.0',
'pytest-cov>=2.7.1',
'pytest-pep8>=1.0.6',
```
The reason for removing these packages is that we are installing the ```"pytest-invenio>=1.4.0",``` package which is installing these packages to keep the tests packages in one place. See [HERE](https://github.com/inveniosoftware/pytest-invenio/blob/798eaa4140dcdfe6e2cb51338915701a993545ee/setup.py#L48)

Reply:
The dependency pytest-pep8 is not satisfied in pytest-invenio. Should it be removed from run-tests.sh?

Reply:
yes you can remove this package, and modify your ```pytest.ini``` file, Check [here](https://github.com/tu-graz-library/invenio-theme-tugraz/blob/master/pytest.ini) for a reference.